README.md

```diff
@@ -261,6 +261,10 @@ https://medium.com/matic-network/understanding-dagger-453d90480c51
 4. A Delegate changes token balance while relaying deposits.
 
   Mitigation would be the same as 2.
+    - Q. How can we know it? Until comparing root receipt with modified data, still blocks can be generated on childchain. And if detection is getting to be late, block amount to be rolled back are somehow big, and double spend can be.
+    - A. We must make detection faster.
```

Comment:
@jdkanani 

When we could make checkpoint period faster by tweaking economical constants, this issue's possible biggest damage would be mitigated by far.

https://maticnetwork.slack.com/files/UAB4M59L0/FB8SM1CAX/screen_shot_2018-06-17_at_22.01.05.png

Reply:
I don't  follow on checkpoint time but let's discuss on call